vaadin-platform-test/collaborationEngineDataDir/ce-license.json

```diff
@@ -0,0 +1 @@
+{"content":{"key":"789f6a8e-a5f3-4eba-80e5-7f062f68ccb0","owner":"integration tests","quota":1000000,"endDate":"2500-12-31"},"checksum":"OizVdCFUmbzkzzXsHc35u9lL2m7vYKCX6gONe3cCRds="}
```

Comment:
Why do we need this?
Is this a valid license, I mean can I copy and paste in my Flow app with CE and it works?

Reply:
Yes, it's a valid license, and technically you can just copy it to your Flow app and CE will work (but sharing one license across apps would be violating the license terms).

We need it here because the tests are running in production mode. CE licensing works like so that in dev mode you can use it for free, but for prod mode, you need to obtain this kind of license from Vaadin that defines the `quota` (how many end users are allowed) and `endDate`.

Reply:
Ok, clear enough. Now my question is can we provide this somehow using deployment vars? like env vars or system properties ?  This repo is public and I don't think it's a good idea to keep this here